### PR TITLE
Enable open registration

### PR DESCRIPTION
**Pull request by silhouettes**
Also tidy up cli output strings. Registrations have not been enabled on the server yet, but will be enabled shortly.

### CODE REVIEW

**Comment by lostintangent** [comment]
LGTM!

File: cli/script/command-parser.ts [hunk 1]
```diff
@@ -231,17 +231,16 @@ var argv = yargs.usage(USAGE_PREFIX + " <command>")
...
-            //.check((argv: any, aliases: { [aliases: string]: string }): any => isValidCommand);  // Report unrecognized, non-hyphenated command category.
-
-        //addCommonConfiguration(yargs);
-    //})
+    .command("register", "Register a new account with the CodePush server", (yargs: yargs.Argv) => {
+        isValidCommandCategory = true;
+        isValidCommand = true;
+        yargs.usage(USAGE_PREFIX + " register")
+            .demand(/*count*/ 1, /*max*/ 2)  // Require one non-optional and one optional argument.
+            .example("register", "Creates a new user account with " + CODE_PUSH_URL)
```
Comment (geof90) [inline cli/script/command-parser.ts]: "the CodePush server"?

**Comment by geof90** [comment]
LGTM!

File: cli/script/command-parser.ts (hunk 1 above)
Comment (lostintangent) [inline cli/script/command-parser.ts]: Agreed!

Comment (silhouettes) [inline cli/script/command-parser.ts]: Agree - missed a spot :)

**Pull request merged**